src/Markdig.Tests/TestFencedCodeBlock.cs

```diff
@@ -0,0 +1,67 @@
+using Markdig.Renderers.Normalize;
+using Markdig.Syntax;
+using NUnit.Framework;
+using System.IO;
+
+namespace Markdig.Tests
+{
+    [TestFixture]
+    public partial class TestFencedCodeBlock
+    {
+        [Test]
+        public void CstInfoParser_ParsesHappyFlow()
```

Comment:
Same no need for Cst term, you can remove it.

Reply:
done